usaspending_api/etl/management/commands/update_total_funding_amount_sql.py

```diff
@@ -0,0 +1,56 @@
+"""
+Creates legal entities and adds them to transaction normalized rows and award rows based on transaction_location_data
+from `create_locations`
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        with connection.cursor() as curs:
+            curs.execute(self.TRANSACTION_FABS_TFA)
+            logger.info('total_funding_amount recalculated. {} successful updates'.format(curs.rowcount))
+
+            curs.execute(self.TRANSACTION_NORMALIZED_TFA)
+            logger.info('total_funding_amount moved to the transaction normalized table. {} successful updates'.format(
+                curs.rowcount))
+
+            curs.execute(self.SUM_TRANSACTION_TFA)
+            logger.info('New sum for total_funding_amount. {} successful updates'.format(curs.rowcount))
+
+    TRANSACTION_FABS_TFA = """
+        update transaction_fabs as tf
+        set total_funding_amount = (
+            tf.federal_action_obligation + tf.non_federal_funding_amount
+        )
+    """
+
+    TRANSACTION_NORMALIZED_TFA = """
+        update transaction_normalized as tn
+        set total_funding_amount=tf.total_funding_amount from transaction_fabs as tf where tf.transaction_id=tn.id;
```

Comment:
`total_funding_amount` should be applicable at the `award` level. It would probably make more sense to have a `funding_amount` field on `transaction_normalized` and then sum that field on related transactions to populated the award's `total_funding_amount`. This way, we're keeping `transaction_fabs` identical to Broker.